src/test/java/Homework18.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework18 extends BaseTest {
+    @Test
+    public void playSong()  {
+        loginToKoel();
+        clickNextSongButton();
+        clickPlaySongButton();
+        verifySoundbarIsVisible();
+        clickPauseSongButton();
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

Reply:
Hi @Berichicko , thank you for your recommendations. I added one more commit with all corrections.
Could you please recomend is it better to use method that returns Web element or void method (line 18-19)?

Reply:
Hello

Please, re-submit homework for review

HINT

void - method: 

click or sendKeys

```
public void enterElement() {
    driver.findElement(By.id("submit-button")).click();
}
```

or

```
public void fillOutFormEmail() {
      driver.findElement(By.id("email")).sendKeys("john@example.com");
  
}

```